### ISSUE TITLE
MHK ranges for Etests

### ISSUE THREAD

**Issue by joheli** [markusrt/NRZMyk#85]
Please note that read points of gradient tests (Etests) are 0.002, 0.003, 0.004, 0.006 etc. (see [bioMérieux package insert](https://techlib.biomerieux.com/wcm/techlib/techlib/documents/docLink/Package_Insert/35904001-35905000/Package_Insert_-_9305056_-_D_-_en_-_Etest_-_AFST_WW.pdf)), corresponding a series where each value equals the previous value multiplied by the squareroot of 2.

E.g. in R:
```R
etest_readpoints <- function(start = 0.002, steps = 20) start * (sqrt(2) ^ (0:steps))
```

**Comment by markusrt** [comment]
@joheli Thanks for the input. I will take a look. @MaxAldejohann also today mentioned some issues regarding the breakpoints. Maybe this is related.

**Comment by joheli** [comment]
Exactly. Please compare entries in above drop-down field with actual read points from gradient tests used in the Würzburg lab. I only use Fluconazol and Anidulafungin in my lab - if you wish I can send you some pictures from those.

**Comment by markusrt** [comment]
Thanks for finding out. You can attach the pictures here. This would be really helpful.

I will update the config accordingly
https://github.com/markusrt/NRZMyk/blob/dea9a16998cdc8d53aaa7a53c0b38161bba2446d/NRZMyk.Client/wwwroot/appsettings.json#L462

@MaxAldejohann will provide steps for the other antimycotics. Issue will be fixed with next release.

**Comment by joheli** [comment]
Ok - please see the images attached (we´re using bioMérieux etests for fluconazole and anidulafungin).

![fluconazole_etest_label](https://user-images.githubusercontent.com/152780/122372224-cbec4080-cf60-11eb-9482-b0dbe339441d.jpg)
![fluconazole_etest_readpoints](https://user-images.githubusercontent.com/152780/122372215-cabb1380-cf60-11eb-978a-02b9d11628fd.jpg)
![anidulafungin_etest_label](https://user-images.githubusercontent.com/152780/122372220-cb53aa00-cf60-11eb-934f-8151d1f27052.jpg)
![anidulafungin_etest_readpoints](https://user-images.githubusercontent.com/152780/122372223-cb53aa00-cf60-11eb-9285-b0433fbda2a1.jpg)

**Comment by markusrt** [comment]
@joheli Update for fluconazole and anidulafungin is realeased with `v.1.0.2106.2220`

Some production data records required an update but mostly trivial, i.e.
- 85 records: 0.12 -> 0.125
- 11 records: 0.3 -> 0.32

However for two of the latter this affected the evaluation as they are now treated as resistant due to break point `Anidulafungin - Candida albicans - v10.0`. Is this expected? Breakpoints as of the initial specification is set to >R -> 0.3

**Comment by markusrt** [comment]
Assigned to @MaxAldejohann as he will provide breakpoints for additional antifungal agents. Additionally he will take a look on the comment regarding margin cases.

**Comment by MaxAldejohann** [comment]
@joheli thx for bringing up this issue. 
@markusrt I can additionally provide MIC values for
-micafungin, caspofungin, voriconazole, posaconzole, ampho B MIC-ranges are equal to that of anidulafungin, so overall i would suggest amending the scale accordingly for all E-tests. 
-concerning the marginal disagreement around cutoff values I would recomend to choose S category in such are cases.
